spec/PhpSpec/Locator/PSR0/PSR0LocatorSpec.php

```diff
@@ -53,6 +53,16 @@ function it_generates_proper_fullSrcPath_even_from_empty_namespace()
         $this->getFullSrcPath()->shouldReturn(dirname(__DIR__).DIRECTORY_SEPARATOR);
     }
 
+    function it_should_not_have_backslash_on_missing_prefix()
+    {
+        $this->beConstructedWith('Cust\Ns', '', dirname(__DIR__), __DIR__);
+
+        $this->getSpecNamespace()->shouldReturn('Cust\Ns\\');
+        $this->getSpecNamespace()->shouldNotReturn('\Cust\Ns\\');
```

Comment:
If it returns the other value you don't need to specify a value it doesn't return